dev/com.ibm.ws.repository.parsers/src/com/ibm/ws/repository/parsers/ParserBase.java

```diff
@@ -141,7 +141,9 @@ public ParserBase() {}
         ND("ND"),
         zOS("zOS"),
         BASE_ILAN("BASE_ILAN"),
-        EARLY_ACCESS("EARLY_ACCESS");
+        EARLY_ACCESS("EARLY_ACCESS"),
+        Open("Open"),
+        Open_Web("Open_Web");
```

Comment:
going from upper case to camel case (ish) is the only change here from the previous attempt at this pull request